ansible/environments/local/group_vars/all

```diff
@@ -26,3 +26,9 @@ limits:
   triggers:
     fires:
       perMinute: 60
+
+# API GW connection configuration
+apigw_auth_user: ""
+apigw_auth_pwd: ""
+apigw_host: "http://{{ groups['apigateway']|first }}:{{apigateway.port.api}}/v1"
```

Comment:
@domdom82 - Yes, this will vary by environment.  In the OW environments it will always refer to a local API GW, in BW other fixed API GWs will be available.